src/components/configuration/DRMConfigurationModal.css

```diff
@@ -0,0 +1,8 @@
+.modal-dark {
+    background-color: #282c34;
+}
+
+hr {
+    color: white;
+    background-color: white;
+}
```

Comment:
Maybe overkill for the project, but i found that .scss files let you do much more things in terms of styling